dgram.js

```diff
@@ -372,6 +368,9 @@ export class Socket extends EventEmitter {
     }
 
     // @XXX(jwerle): @heapwolf why is this happening in a `send()` call?
+    //
+    // @jwerle it's from the node.js source code - https://github.com/nodejs/node/blob/main/lib/dgram.js#L645
```

Comment:
okay makes sense!

Reply:
Where did you see the lazy binding? I wonder about that.